src/entity/Human.php

```diff
@@ -211,6 +211,14 @@ public function getXpManager() : ExperienceManager{
 		return $this->xpManager;
 	}
 
+	public function getXpSeed() : int{
+		return $this->xpSeed;
+	}
+
+	protected function generateXpSeed() : void{
```

Comment:
As the method says, it need to generate, so return a value and not setting?

Reply:
Yes, that's me, but what I meant was a function to generate a seed, but not a function to define the seed.